src/picongpu/include/plugins/output/images/PngCreator.hpp

```diff
@@ -186,9 +180,6 @@ namespace picongpu
             if ((scale_x != float_X(1.0)) || (scale_y != float_X(1.0)))
```

Comment:
You are right I need to handle the case with the scaling to the cell size and without